sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -46,9 +46,9 @@ grammar SqlBase;
   }
 
   /**
-   * When true, ANSI SQL parsing mode is enabled.
+   * When true, use ANSI SQL standard keywords.
```

Comment:
`When true, the behavior of keywords follows ANSI SQL standard`

Reply:
Copy, done in 5a4b2ea